qiskit/aqua/algorithms/minimum_eigen_solvers/vqe.py

```diff
@@ -163,10 +154,13 @@ def __init__(self,
         self._eval_time = None
         self._eval_count = 0
 
+        super().__init__(var_form=var_form,
```

Comment:
Yes, because the `VQAlgorithm` uses the `var_form` setter which needs `self._operator` to be defined

Reply:
In the code currently in master I set only the instance var in VQAlgorithm in its constructor so as to avoid this type of issue that the parent ends up calling a potentially overridden method in a derived class. It makes creating new derived classes more brittle. I see it was changed in the Optimization PR but I do not understand why since to me it makes things more tricky for the subclass